source/includes/v2/_sources.md

```diff
@@ -1,32 +1,33 @@
 # Sources
 
-> Retrieve a list of available sources on `public` domain
+> Retrieve a list of available sources on the `documentation-resource` domain
 
 ```shell
-curl 'https://examples.opendatasoft.com/api/v2/'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/'
 ```
 
 > The above command returns
 
 ```json
 {
-	"links": [{
-			"href": "https://examples.opendatasoft.com/api/v2",
-			"rel": "self"
-		},
-		{
-			"href": "https://examples.opendatasoft.com/api/v2/catalog",
-			"rel": "catalog"
-		},
-		{
-			"href": "https://examples.opendatasoft.com/api/v2/monitoring",
-			"rel": "monitoring"
-		},
-		{
-			"href": "https://examples.opendatasoft.com/api/v2/opendatasoft",
-			"rel": "opendatasoft"
-		}
-	]
+    "links": [
```

Comment:
Same as above.

Reply:
If we remove links here, the endpoint returns a blank response. Is that right?

Reply:
Yes indeed, it doesn't make sense to remove links here. I'm taking the insight for squad-engine, this endpoint may be an exception as links _are_ the expected objects. Maybe you can just keep `links` in this example for now?

Reply:
OK, I'll leave this example as it is.